contrib/cni/10-crio-bridge.conflist

```diff
@@ -1,24 +1,20 @@
 {
-  "cniVersion": "1.0.0",
-  "name": "crio",
-  "plugins": [
-    {
-      "type": "bridge",
-      "bridge": "cni0",
-      "isGateway": true,
-      "ipMasq": true,
-      "hairpinMode": true,
-      "ipam": {
-        "type": "host-local",
-        "routes": [
-            { "dst": "0.0.0.0/0" },
-            { "dst": "::/0" }
-        ],
-        "ranges": [
-            [{ "subnet": "10.85.0.0/16" }],
-            [{ "subnet": "1100:200::/24" }]
-        ]
-      }
-    }
-  ]
-}
+	"cniVersion": "1.0.0",
+	"name": "crio-bridge",
+	"type": "bridge",
+	"bridge": "cni0",
+	"isGateway": true,
+	"ipMasq": true,
+  	"hairpinMode": true,
+	"ipam": {
+		"type": "host-local",
+		"routes": [
+			{ "dst": "0.0.0.0/0" },
+      { "dst": "::/0" }
+		],
+    "ranges": [
+      [{ "subnet": "10.85.0.0/16" }],
+      [{ "subnet": "1100:200::/24" }]
+    ]
+	}
+}
```

Comment:
This does not appear to be the latest format per the CNI spec version 1.1.0. Also, the JSON is not well-formatted.